pkg/api/types.go

```diff
@@ -367,6 +370,25 @@ type GCEPersistentDiskVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// A ISCSI Disk can only be mounted as read/write once.
+type ISCSIVolumeSource struct {
+	// Required: iSCSI target portal
+	// the portal is either an IP or ip_addr:port if port is other than default
```

Comment:
from wiki:
iSCSI uses TCP (typically TCP ports 860 and 3260) for the protocols itself.